gp-templates/translation-row.php

```diff
@@ -139,14 +139,18 @@
 					<?php echo display_status( $t->translation_status ); ?>
 					<?php if ( $t->translation_status ): ?>
 						<?php if ( $can_approve_translation ) : ?>
-							<?php if ( $t->translation_status != 'current' ): ?>
+							<?php if ( 'current' !== $t->translation_status ) : ?>
 							<button class="approve" tabindex="-1" data-nonce="<?php echo esc_attr( wp_create_nonce( 'update-translation-status-current_' . $t->id ) ); ?>"><strong>+</strong> <?php _e( 'Approve', 'glotpress' ); ?></button>
 							<?php endif; ?>
-							<?php if ( $t->translation_status != 'rejected' ): ?>
+							<?php if ( 'rejected' !== $t->translation_status ) : ?>
 							<button class="reject" tabindex="-1" data-nonce="<?php echo esc_attr( wp_create_nonce( 'update-translation-status-rejected_' . $t->id ) ); ?>"><strong>&minus;</strong> <?php _e( 'Reject', 'glotpress' ); ?></button>
 							<?php endif; ?>
+							<?php if ( 'fuzzy' !== $t->translation_status ) : ?>
+							<button class="fuzzy" tabindex="-1" data-nonce="<?php echo esc_attr( wp_create_nonce( 'update-translation-status-fuzzy_' . $t->id ) ); ?>"><strong>~</strong> <?php _e( 'Fuzzy', 'glotpress' ); ?></button>
+							<?php endif; ?>
 						<?php elseif ( $can_reject_self ): ?>
 							<button class="reject" tabindex="-1" data-nonce="<?php echo esc_attr( wp_create_nonce( 'update-translation-status-rejected_' . $t->id ) ); ?>"><strong>&minus;</strong> <?php _e( 'Reject Suggestion', 'glotpress' ); ?></button>
+							<button class="fuzzy" tabindex="-1" data-nonce="<?php echo esc_attr( wp_create_nonce( 'update-translation-status-fuzzy_' . $t->id ) ); ?>"><strong>~</strong> <?php _e( 'Fuzzy', 'glotpress' ); ?></button>
```

Comment:
If a can reject a translation they should probably be able to set it fuzzy which is a less significant change in status.